src/Nullable/Nullable.ts

```diff
@@ -1,8 +1,8 @@
 export type NotNull<T> = T extends null ? never : T;
 
-export type Nullable<T> = T | null;
+export type Nullable<T> = NotNull<T> | null;
```

Comment:
By the current implementation (alpha-0), the following case would be error:

```typescript
function bar<T>(input: T): Nullable<T> {
  if (Math.random() > 0.5) {
    return input; // <- compile error.
    // Type 'T' is not assignable to type 'Nullable<T>'.
    // Type 'T' is not assignable to type 'NotNull<T>'.ts(2322
  }
  return null;
}
```

We need to change to make more soft.